src/liboslexec/wide/wide_opdictionary.cpp

```diff
@@ -145,12 +145,12 @@ __OSL_MASKED_OP(dict_next)(void* bsg_, void* wout, void* wNodeID,
 
 
 OSL_BATCHOP int
-__OSL_OP(dict_value)(void* bsg_, int nodeID, void* attribname, long long type,
-                     void* data)
+__OSL_OP(dict_value)(void* bsg_, int nodeID, ustring_pod attribname,
+                     long long type, void* data)
 {
     auto* bsg = reinterpret_cast<BatchedShaderGlobals*>(bsg_);
     return bsg->uniform.context->dict_value(nodeID, USTR(attribname),
-                                            TYPEDESC(type), data);
+                                            TYPEDESC(type), data, false);
```

Comment:
Add comment on `false `argument identifying the name of parameter `/* treat_ustrings_as_hash */`